### DISCUSSION TITLE
Select component: bredde

### DISCUSSION THREAD

**Discussion by ivnai** [fremtind/jokul#3644] [Spørsmål]
Hei, er det mulig å enkelt sette forskjellige bredde på Select input vs Dropdown del?
Vi på Flyt har behov f.eks:

Gamle design
<img width="465" alt="Screenshot 2023-08-14 at 13 10 12" src="https://github.com/fremtind/jokul/assets/93194579/49efbbf7-2411-48b5-9833-d1055e795c3d">

Nytt design
<img width="566" alt="Screenshot 2023-08-14 at 13 10 20" src="https://github.com/fremtind/jokul/assets/93194579/cbfd3b69-22a8-42ae-876d-6174508160a0">

Eller casen her (komponenten er Flyt sin men vi streber å erstatte alle "lokale" med Jøkul komponentene
<img width="695" alt="Screenshot 2023-08-14 at 13 10 32" src="https://github.com/fremtind/jokul/assets/93194579/00266dc7-fb20-4c5d-a1dc-a6e38d86bcbc">

..og mange flere

**Comment by leon-ho** [comment]
Diskutert i Designsystemforum 22.08.2023 - Generell om at det kan være lurt å støtte bredere liste i forhold til inputfeltet.

(Illustrasjonsskisse)
![image](https://github.com/fremtind/jokul/assets/1818882/f39de626-e4dd-4174-a40e-76bccc850bed)
